src/peft/tuners/lora/layer.py

```diff
@@ -1113,6 +1125,16 @@ def __repr__(self) -> str:
         return "lora." + rep
 
 
+class Conv2d(ConvNd):
```

Comment:
Similar argument to what I mentioned above for IA³ and for `self.conv_fn`.

Reply:
I don't follow; do you mean a sanity check on the self._kernel_dim? I added those in the most recent commits.